packages/form-js-viewer/assets/form-js.css

```diff
@@ -178,6 +178,8 @@
   width: auto !important;
   min-width: 34px;
   text-align: center;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
Works in the viewer, but doesn't truncate in the editor.

![image](https://user-images.githubusercontent.com/9433996/211556053-7f77eda9-01ed-47a7-8076-e1a0b1c4bca5.png)

Can you verify that? Maybe I missed something.